content/en/docs/Writing policies/variables.md

```diff
@@ -326,7 +326,7 @@ You can also fetch image properties of all containers for further processing. Fo
 
 ## Inline Variables
 
-Variables may be defined in a `context` for consumption by Kyverno rules. This can be as simple as a static value or an object. The below sets a context variable with a value of `foo`.
+Variables may be defined in a `context` for consumption by Kyverno rules. This can be as simple as a static value, another variable, or a nested object. The below sets a context variable with a value of `foo`.
```

Comment:
Another important thing to note is that variables can be redefined by using the same variable name. The last value that is set is used.

You can also reference the same variable when redefining its value.

Reply:
Example https://github.com/kyverno/kyverno/blob/fe3c12628c9356eadf25c48717db9b9bf9e9eac8/test/cli/test/context-entries/policies.yaml#L125

Reply:
Captured in https://github.com/kyverno/website/pull/548/commits/3f68311543a3e77b2d730b7570d5b00b8b100429